webapp/publisher/snaps/logic.py

```diff
@@ -50,6 +50,29 @@ def get_snaps_account_info(account_info):
     return user_snaps, registered_snaps
 
 
+def get_snap_names_by_ownership(account_info):
+    """Get list of snaps names user is collaborator of
+
+    :param account_info The account informations
+
+    :return A list of owned snaps names
```

Comment:
syntax:

```
:param account_info: blabla

:returns: bla bla
```

Reply:
I think I copied it from somewhere... but will fix.